commons-test/src/main/java/org/infinispan/commons/test/TestSuiteProgress.java

```diff
@@ -16,63 +22,77 @@
 
    private static final Logger log = Logger.getLogger(TestSuiteProgress.class);
 
-   private static AtomicInteger failed = new AtomicInteger(0);
-   private static AtomicInteger succeeded = new AtomicInteger(0);
-   private static AtomicInteger skipped = new AtomicInteger(0);
+   private AtomicInteger failed = new AtomicInteger(0);
+   private AtomicInteger succeeded = new AtomicInteger(0);
+   private AtomicInteger skipped = new AtomicInteger(0);
+   private final PrintStream out;
+   private final boolean useColor;
+
+   public TestSuiteProgress() {
+      useColor = System.getenv("JENKINS_HOME") == null;
```

Comment:
I wonder if this is really necessary... jansi uses [`isatty`](http://man7.org/linux/man-pages/man3/isatty.3.html) to avoid writing the escape codes when output is redirected.

Reply:
Actually, I see that `isatty` check is in `AnsiConsole`, and you're not using it... still, when I run the build from my Python script (which uses `Popen` to parse the output and set the test count in the title), the colours just disappear.